addons/product/pricelist_view.xml

```diff
@@ -87,38 +29,54 @@
+                            <field name="product_id" attrs="{'invisible':[('applied_on', '!=', '0')]}" string="Product Variant"/>
+                        </group>
+                        <group>
+                            <field name="min_quantity"/>
+                            <field name="date_start"/>
+                            <field name="date_end"/>
+                        </group>
                     </group>
                     <separator string="Price Computation"/>
+                    <group>
+                        <group>
+                            <label for="compute_price" string="Compute Price"/>
+                            <div>
+                                <field name="compute_price" widget="radio"/>
+                                <field name="currency_id" invisible="1"/>
+                                <div attrs="{'invisible':[('compute_price', '!=', 'fixed')]}">
+                                    <field name="fixed_price" nolabel= "1" widget='monetary' options="{'currency_field': 'currency_id'}"/></div>
+                                <div attrs="{'invisible':[('compute_price', '!=', 'percentage')]}">
+                                    <field name="percent_price" nolabel="1" class="oe_inline"/>%%
+                                </div>
+                            </div>
+                        </group>
+                    </group>
                     <div class="oe_grey" groups="product.group_uom">
                         <p>The computed price is expressed in the default Unit of Measure of the product.</p>
                     </div>
-                    <group col="4">
-                        <field name="base"/>
-                        <field name="base_pricelist_id" attrs="{'invisible':[('base', '!=', -1)],'required': [('base','=', -1)], 'readonly': [('base','!=', -1)]}"/>
-                    </group>
-                    <group col="6" colspan="5">
-                        <label string="New Price ="/>
-                        <label string="Base Price"/>
-                        <label string="* ( 1 + "/>
-                        <field name="price_discount" nolabel="1"/>
-                        <label string=" ) + "/>
-                        <field name="price_surcharge" nolabel="1"/>
-                        <newline/>
-                        <field name="price_round" string="Rounding Method"/>
-                        <field name="price_min_margin" string="Min. Margin"/>
-                        <field name="price_max_margin" string="Max. Margin"/>
+                    <group attrs="{'invisible':[('compute_price', '!=', 'formula')]}">
+                        <group col="6" colspan="6">
+                            <field name="base" colspan="6"/>
+                            <label string="New Price ="/>
+                            <label string="Public Price  -  " attrs="{'invisible':[('base', '=', 'pricelist')]}"/>
```

Comment:
better to use this attrs : "{'invisible':[('base', 'not in', ('list_price','standard_price'))]}